resources/assets/sass/components/_logo.scss

```diff
@@ -0,0 +1,16 @@
+.c-logo {
+
+	display: block;
+	height: 132px;
+	width: 279px;
+	background: url('#{$path-svg}tripee-logo.svg') no-repeat center center;
```

Comment:
Remove url part, lets move the setting the logo background image to Blade side (dynamic logos etc).